packages/cli/src/commands/deploy/args.ts

```diff
@@ -68,7 +68,7 @@ export const help = () => `
     -m, --meta                     Add metadata for the deployment (e.g.: ${chalk.dim(
       '`-m KEY=value`'
     )}). Can appear many times.
-    -C, --no-clipboard             Do not attempt to copy URL to clipboard
+    -C, --clipboard                Copy URL to clipboard after deployment
```

Comment:
This should be removed

Also the docs need to be removed here https://vercel.com/docs/cli#introduction/unique-options/no-clipboard